### ISSUE TITLE
1.19.4 game crashes on start up

### ISSUE THREAD

**Issue by Thunderbird4076** [Chocohead/OptiFabric#1077]
When I launch the game it doesn't give me an error code or anything all it does is crash and gives me the exit code 1.

**Comment by PhantomPhoenix2016** [comment]
yes same thing happens to me, but I don't know how to fix it. Someone help please

**Comment by goose-vii** [comment]
Same here as well
*ETA: I'm using the 1.19.4 OptiFabric and Optifine (respectively)
and have tried running it with no other mods or RPs, to no success unfortunately; still just crashes super hard.

**Comment by desperados-1** [comment]
Same thing

**Comment by desperados-1** [comment]
solution - #1065
